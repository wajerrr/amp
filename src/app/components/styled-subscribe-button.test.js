import React from 'react';
import renderer from 'react-test-renderer';
import StyledSubscribeButton from './styled-subscribe-button';

describe('StyledSubscribeButton', () => {
  it('should match snapshot', () => {
    const tree = renderer.create(<StyledSubscribeButton />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
