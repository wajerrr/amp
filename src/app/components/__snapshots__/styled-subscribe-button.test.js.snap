// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledSubscribeButton should match snapshot 1`] = `
<a
  className="css-856nem-StyledLink e1il8zx80"
  href="https://econ.st/2bs1gXz"
  role="button"
>
  Subscribe
</a>
`;
